src/Dialogs/Message.cpp

```diff
@@ -153,3 +153,51 @@ ShowMessageBox(const TCHAR *text, const TCHAR *caption, unsigned flags)
 
   return res;
 }
+
+
+
+void
+ShowLockBox()
+{
+  SingleWindow &main_window = UIGlobals::GetMainWindow();
+
+  const unsigned button_height = Layout::GetMinimumControlHeight();
+  const unsigned button_width = button_height;
+
+  // Create dialog
+  WindowStyle style;
+  style.ControlParent();
+
+  const DialogLook &dialog_look = UIGlobals::GetDialogLook();
+
+  const PixelSize root_size = main_window.GetSize();
+  
+  const int dialog_x = (root_size.cx - button_width) / 4;
+  const int dialog_y = (root_size.cy - button_height) / 4 * 3;
+
+  PixelRect form_rc;
+  form_rc.left = dialog_x;
+  form_rc.top = dialog_y;
+  form_rc.right = dialog_x + button_width;
+  form_rc.bottom = dialog_y + button_height;
+
+  WndForm wf(main_window, dialog_look, form_rc, "", style);
+
+  ContainerWindow &client_area = wf.GetClientAreaWindow();
+  
+  PixelRect button_rc;
+  button_rc.left = 0;
+  button_rc.top = 0; 
+  button_rc.right = button_width;
+  button_rc.bottom = button_height;
+
+  // Create button
+  WindowStyle button_style;
+  button_style.TabStop();
+  
+  const TCHAR * caption = "U";
+  const Button button(client_area, dialog_look.button, caption, button_rc, button_style, wf, IDOK);
+
+  // Show MessageBox
```

Comment:
Typical example of a useless code comment. All it does is describe something that is obvious to anybody reading the following line.